src/components/SnapshotDetailView.jsx

```diff
@@ -135,6 +135,7 @@ export class SnapshotDetailView extends React.PureComponent {
               datasetsCount={snapshot.source.length}
               features={features}
               handleFilterDatasets={this.handleFilterDatasets}
+              searchString={''}
```

Comment:
I think you can just have
```suggestion
              searchString=""
```